__tests__/app.test.js

```diff
@@ -33,3 +34,60 @@ describe("ANY request to invalid path", () => {
    });
  });
 });
+describe("/api", () => {
+ test("should respond with a 200 status code", () => {
+  return request(app).get("/api").expect(200);
+ });
+ test("should respond with an object with all available endpoints on the API", () => {
+  return request(app)
+   .get("/api")
+   .then(({ body }) => {
+    expect(Object.keys(body.endpoints).length).toBe(
+     Object.keys(endPoints).length
+    );
+    for (let key in body.endpoints) {
+     expect(typeof body.endpoints[key].description).toBe("string");
+     expect(typeof body.endpoints[key].queries).toBe("object");
+     expect(typeof body.endpoints[key].exampleResponse).toBe("object");
+     expect(typeof body.endpoints[key].requestBodyFormat).toBe("object");
```

Comment:
One suggestion from me here is that you could `const / require` in the file, and then assert that your response looks exactly like the contents of the file using `toEqual`